CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Changelog
 
+## v1.0.0-rc.9 (12/10/2020)
+
+## Breaking changes
+
+- #199 Integrate Ckeditor into Property View
```

Comment:
don't we link to the PRs like in dsp-js?

Reply:
I think this was https://dasch.myjetbrains.com/youtrack/issue/DSP-682, the PR's description contains additional information about the breaking change

Reply:
let me check...

Reply:
I had to update some titles of already closed PRs and update the "Next Release" draft. Now the changelog file should be good